client/src/containers/barMainContainer.jsx

```diff
@@ -0,0 +1,22 @@
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { deletePage, sortPages } from '../actions/index';
+
+import BarMain from '../components/barmain';
+
+function mapStateToProps(state) {
+  return {
+    pages: state.barMain.pages,
```

Comment:
Are these being called the same way as pagelist? Is this container necessary? It seems the same thing could be accomplished by calling the same props from the pageList part of store.